src/components/common.js

```diff
@@ -54,39 +54,40 @@ const styles = {
   }
 }
 
```

Comment:
--- Beginning of unrelated reformatter changes ---

Reply:
What was the motivation for re-formatting this?

Reply:
I have IntelliJ do the reformatting when I'm unsure about how to format code that I'm changing. I don't have a way to selectively apply that reformatting on a piece of a file. I hate to increase the PR diff but not reformatting now means I'd have to manually unapply reformatting every time I need to run the reformatter.

Reply:
@kyuksel you can select just a portion of the text and choose "Reformat Code".

![image](https://user-images.githubusercontent.com/484484/173448148-d350d1f3-8894-4b27-860b-ab813d7d1fec.png)

Reply:
Oh wow -- so many pearls, you two 🤩